internal/abci/abci.go

```diff
@@ -281,35 +302,14 @@ func NewAbciApp(ctx context.Context, cfg *AbciConfig, snapshotter SnapshotModule
 		app.consensusParams.Votes.MaxVotesPerTx = networkParams.MaxVotesPerTx
 	}
 
-	migrationStatus := app.migrator.MigrationStatus()
-	app.log.Infof("Migration status: %s", migrationStatus.String())
-
-	networkParams.MigrationStatus = migrationStatus
-
-	// if the migration is in progress, we need to set the network params to reflect that
-	// networkParams.InMigration = migrationStatus == types.MigrationInProgress || migrationStatus == types.MigrationCompleted
-
-	// if the migration is completed, we need to halt the network
-	// networkParams.MigrationCompleted = migrationStatus == types.MigrationCompleted
-
-	var migrationParams *common.MigrationContext
-	startHeight := app.consensusParams.Migration.StartHeight
-	endHeight := app.consensusParams.Migration.EndHeight
-
-	if startHeight != 0 && endHeight != 0 {
-		// set this only if "migrate_from" is configured
-		migrationParams = &common.MigrationContext{
-			StartHeight: startHeight,
-			EndHeight:   endHeight,
-		}
-	}
-
 	app.chainContext = &common.ChainContext{
 		ChainID:           cfg.ChainID,
 		NetworkParameters: networkParams,
 		MigrationParams:   migrationParams,
 	}
 
+	migrator.SetMigrationStatus(app.chainContext.NetworkParameters.MigrationStatus)
```

Comment:
Small issue is that the `NetworkParameters.MigrationStatus` field in `common/common.go` is out of date.  I think those possible values shouldn't be listed there anyway since they'll never get updated there.